### ISSUE TITLE
Examples and Courses doesn't work on a Website

### ISSUE THREAD

**Issue by jcubic** [AlaSQL/alasql#1570]
It seems that the main menu "Examples" and "Courses" is something left from an old website or something that got deleted. Because they don't work.

Another issue is that:

https://mc.yandex.ru/metrika/tag.js

file doesn't load because of an invalid Cert.

**Comment by JuanPabloDiaz** [comment]
Yeah, those two links are not working in the main menu and also in the footer.
